Source/WebGPU/WebGPU/ComputePassEncoder.mm

```diff
@@ -82,11 +82,11 @@
     uint32_t bindGroup;
 };
 using EntryMap = HashMap<uint64_t, EntryUsageData, DefaultHash<uint64_t>, WTF::UnsignedWithZeroKeyHashTraits<uint64_t>>;
-using EntryMapContainer = HashMap<void*, EntryMap>;
+using EntryMapContainer = HashMap<const void*, EntryMap>;
```

Comment:
For a followup: I have caused bugs in the recent past with raw pointers as map keys and accidental re-use of map entries due to not removing entries when the object goes away. If we could use weak pointer keyed map here, I think we should prefer that.

Reply:
That is a good idea, I will file a bug. Just recently learned we have WeakHashMap.

Reply:
https://bugs.webkit.org/show_bug.cgi?id=269620